interpreter/cling/include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -241,6 +251,13 @@ namespace cling {
     return collectionPrinterInternal::printValue_impl(obj);
   }
 
+#if __cplusplus >= 201703L
+  // Specialization for std::filesystem::path
+  template <> inline std::string printValue(const std::filesystem::path* obj) {
+    return collectionPrinterInternal::printValue_impl(obj);
+  }
```

Comment:
Do we need this indirection or can we just declare `std::string printValue(const std::filesystem::path* value)` as we do for primitive data types and `std::string` etc. above?

Reply:
I think we could live without unless there is a reason that I missed.

Reply:
I agree, the simple overload should be enough